lib/fastlane/plugin/wpmreleasetoolkit/helper/app_size_metrics_helper.rb

```diff
@@ -0,0 +1,95 @@
+require 'json'
+require 'net/http'
+require 'zlib'
+
+module Fastlane
+  module WPMRT
+    # A helper class to build an App Size Metrics payload and send it to a server (or write it to disk)
+    #
+    # The payload generated (and sent) by this helper conforms to the API for grouped metrics described in
+    # https://github.com/Automattic/apps-metrics
+    #
+    class AppSizeMetricsHelper
+      # @param [Hash] group_meta Metadata common to all the metrics. Can be any arbitrary set of key/value pairs.
+      #
+      def initialize(group_meta = {})
```

Comment:
Nitpick: Would you consider being more verbose use "metadata" over "meta"?

I think "meta" is clear enough, but "metadata" is clearer 😄

Reply:
Addressed in e8acd9d328088d94a36692a7c3bb25d227adb5a5